css/index.css

```diff
@@ -342,3 +331,88 @@ footer {
   color: rgba(187, 14, 14, 0.815);
   font-weight: 600;
 }
+.tabs .devs-container {
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  width: 600px;
+  margin: 30px auto 20px;
+  font-size: 2rem;
+}
+@media (max-width: 800px) {
+  .tabs .devs-container {
+    width: 500px;
+  }
+}
+@media (max-width: 500px) {
+  .tabs .devs-container {
+    width: 300px;
+    height: 125px;
+    justify-content: space-around;
+  }
+}
+.tabs .devs-container .tab {
+  text-align: center;
+  border: solid 3px rgba(187, 14, 14, 0.815);
+  border-radius: 8px;
+  width: 90px;
```

Comment:
90px might not be wide enough. I've noticed your name escapes the box when I've got the about page open. It might also be nice to put a cursor: pointer in here.